zircbot/__init__.py

```diff
@@ -33,6 +33,8 @@
 from twisted.internet import reactor, protocol
 from twisted.python import log
 from twisted.words.protocols import irc
+from zircbot.backends import trello
```

Comment:
I would prefer plugins rather than backends. As there is no database or whatever backend involved here.